extension/css/cryptup.css

```diff
@@ -2398,7 +2408,7 @@ body.attachment .file-download-right-click-link {
   text-decoration: none;
 }
 
-.file-download-right-click-link {
+#attachment-preview-download .file-download-right-click-link {
```

Comment:
@tomholub I made a mistake in this line in https://github.com/FlowCrypt/flowcrypt-browser/pull/2778 which caused the regression when downloading attachments. The font color of the `Right-click here and choose 'Save Link As…' to save encrypted file` link was white on white background which made it invisible for users :/

If possible, please release a new patch version, this PR fixed the issue.

Reply:
Got it - will do!